core/src/main/java/hudson/PluginWrapper.java

```diff
@@ -253,7 +253,7 @@ public void setOptionalDependents(@NonNull Set<String> optionalDependents) {
      */
     @Deprecated
     public void setOptionalDependants(@NonNull Set<String> optionalDependents) {
-        setOptionalDependents(dependents);
+        setOptionalDependents(optionalDependents);
```

Comment:
Very nice catch!  As far as I can tell from a search of the jenkinsci GitHub organization, there are no consumers of this deprecated method.  The code change is correct.